scripts/setup_fvp_optee.sh

```diff
@@ -180,6 +180,8 @@ export CROSS_COMPILE=$DST_AARCH64_NONE_GCC/bin/aarch64-none-elf-
 cd $DST_KERNEL
 
 if [ ! -f ".config" ]; then
+	sed -i '/config ARM64$/a\
+	select DMA_SHARED_BUFFER' arch/arm/Kconfig
```

Comment:
I agree. But I could not find any other automatic way to include DMA_BUF. I'm opened to any proposal :)